dynamic_values/config_modules/service_authorizations/service_authorizations.tf

```diff
@@ -55,6 +59,22 @@ module "kms_to_block_storage" {
   ]
 }
 
+# workaround for https://github.ibm.com/GoldenEye/issues/issues/6709
```

Comment:
use the internal issue https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/issues/645

Reply:
Done.